utils/ansible.py

```diff
@@ -0,0 +1,269 @@
+import fauxfactory
+from git import Repo
+import os
+import subprocess
+import yaml
+from shutil import copy, copyfile, rmtree
```

Comment:
REQUIRED: we have some rules for import statements.you  can read them [here](http://cfme-tests.readthedocs.io/guides/dev_guide.html)
Please break up imports into 3 sections - standard library, third-party, local application and sort them alphabetically at least.